codec/decode.go

```diff
@@ -2048,9 +2054,26 @@ func (d *Decoder) Decode(v interface{}) (err error) {
 	return
 }
 
+// The default starting value of remainingDepth.
+//
+// TODO: Make this configurable.
+const defaultMaxDepth = 100
```

Comment:
lowered this, don't think we need more than 100

Reply:
Can we put this in the decoder handle?

Reply:
Will do, although we'll still need a default value.

Reply:
Done.